Data/Sys/GameSettings/RM8E01.ini

```diff
@@ -137,220 +153,243 @@ de000000 80008180
+1409bfcc 60000000
+1409bff8 60000000
+1409c00c 60000000
+e0000000 80008000
+*Taunt during your turn.
+*Also removes the cap on how fast you can taunt.
+
+$QOL - Increased Text Display [Nora]
+0405DED4 60000000
+*Text is displayed instantly.
+
+$QOL - Remove Explanations [Rain]
+82200001 80228760
+86300001 ffffbfff
+84200001 80228760
+*Remove Minigame Explanations in all minigames.
+
+$Board: DK's Treetop Temple - Free Cannon [gamemasterplc]
+48000000 800030c8
+de000000 80008180
+3001a2dc 387f00c8
+1401a2d8 60000000
+1401a160 2c040000
+e0000000 80008000
+*Cannon usage is free in the board DK's Treetop Temple
+
+$Board: Goomba's Booty Boardwalk - Remove Dolphin Shuttles [gamemasterplc]
+2057bc40 437f0000
+0457bc3c 42700000
+e2000001 80008000
+*A long teadious path forms in the board Goomba's Booty Boardwalk.
+
+$Board: King Boo's Haunted Hideaway - Make the Whomp Toll Free [gamemasterplc]
+48000000 800030c8
+de000000 80008180
+3003f0e0 387f01c0
+1403f0dc 60000000
+1403f04c 2c03000a
+e0000000 80008000
+*Whomp toll is free in the board King Boo's Haunted Hideaway
+
+$Board - King Boo's Haunted Hideaway: Map Always Revealed [gamemasterplc]
+042287bc ffffffff
+*King Boo's Haunted Hideaway becomes not that secret.
 
-$Netplay Force Player 1 to CPU
+$Extra - Disable Music [gamemasterplc]
+042CE168 00000001
+0404D9FC 60000000
+*Disables all in-game music.
+
+$Mechanics - Cashzap Candy Steals Coins [gamemasterplc]
 48000000 800030c8
 de000000 80008180
-301a0218 4bffb149
-141a0204 48000030
+301020d8 80bf01a8
+d21020c4 00000005
+3dc08023 89ee876a
+39ce82f8 1def0118
+7dcf7214 a1ee0026
+7def1a14 b1ee0026
+907f01a8 00000000
 e0000000 80008000
 48000000 800030c8
 de000000 80008180
-301a96d4 4bffb149
-141a96c0 48000030
+3010d220 80bf01a8
+d210d20c 00000005
+3dc08023 89ee876a
+39ce82f8 1def0118
+7dcf7214 a1ee0026
+7def1a14 b1ee0026
+907f01a8 00000000
 e0000000 80008000
 48000000 800030c8
 de000000 80008180
-301c6194 4bffb149
-141c6180 48000030
+30128f9c 80bf01a8
+d2128f88 00000005
+3dc08023 89ee876a
+39ce82f8 1def0118
+7dcf7214 a1ee0026
+7def1a14 b1ee0026
+907f01a8 00000000
 e0000000 80008000
 48000000 800030c8
 de000000 80008180
-301a4954 4bffb149
-141a4940 48000030
+30106b8c 80bf01a8
+d2106b78 00000005
+3dc08023 89ee876a
+39ce82f8 1def0118
+7dcf7214 a1ee0026
+7def1a14 b1ee0026
+907f01a8 00000000
 e0000000 80008000
 48000000 800030c8
 de000000 80008180
-301a24e8 4bffb149
-141a24d4 48000030
+30104498 80bf01a8
+d2104484 00000005
+3dc08023 89ee876a
+39ce82f8 1def0118
+7dcf7214 a1ee0026
+7def1a14 b1ee0026
+907f01a8 00000000
 e0000000 80008000
 48000000 800030c8
 de000000 80008180
-301a6ba0 4bffb149
-141a6b8c 48000030
+30109890 80bf01a8
+d210987c 00000005
+3dc08023 89ee876a
+39ce82f8 1def0118
+7dcf7214 a1ee0026
+7def1a14 b1ee0026
+907f01a8 00000000
 e0000000 80008000
-*This cheat makes it so the player on port 1 will be turned into a computer. This can be useful for games when the player with port 1 drops.
+*Cashzap Candy steals coins rather then destroying.
```

Comment:
```suggestion
*Cashzap Candy steals coins rather then destroying them.
```

Reply:
rather th**a**n

Reply:
It's a super minor grammar error so I think we can let it slide for now instead of possibly messing up the commit history.